pkg/volume/csi/csi_attacher.go

```diff
@@ -139,6 +146,11 @@ func (c *csiAttacher) WaitForAttach(spec *volume.Spec, _ string, pod *v1.Pod, ti
 
 	attachID := getAttachmentName(source.VolumeHandle, source.Driver, string(c.plugin.host.GetNodeName()))
 
+	if c.plugin.volumeAttachmentInformerHook != nil {
```

Comment:
I didn't leave a comment here because this method watches for an attach that was already created by the caller.